src/main/resources/application.properties

```diff
@@ -15,9 +39,9 @@ spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 
 ## PostgreSQL
 spring.datasource.platform=postgres
-spring.datasource.url=${POSTGRES_CONNECTION_URL}
+spring.datasource.url=jdbc:postgresql://localhost:5432/updated-postgres
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
-#spring.datasource.username=${POSTGRESUSERNAME}
-#spring.datasource.password=${POSTGRESPASSWORD}
+spring.datasource.username=${POSTGRESUSERNAME}
+spring.datasource.password=${POSTGRESPASSWORD}
```

Comment:
i think we should return this to use the existing azure db

Reply:
yep. reverted the app.props change.

Reply:
this will likely fail though since we don't have all those fields in the azure db yet

Reply:
yes. so we need to first update the azure table instance